src/pages/Landing.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useOidc } from '@axa-fr/react-oidc';
+import { TableCellsIcon, UserIcon } from '@heroicons/react/24/solid';
+import { getLogger } from '../logger';
+import { Panel, PanelBody } from '../components/Panel';
+import { Button } from '../components/Button';
+import './Landing.css';
+
+const logger = getLogger('<Landing>');
+
+const Landing = () => {
+  const { login, isAuthenticated } = useOidc();
+
+  return (
+    <Panel
+      fill={false}
+      className="landing-panel"
+    >
+      <PanelBody>
+        <h3 style={{ fontWeight: '600' }}>Welcome to the Philanthropy Data Commons pilot.</h3>
+        <p>
+          This data viewer displays the data in the Philanythropy Data Commons pilot.
+          {' '}
+          <a
+            href="https://philanthropydatacommons.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Read more about the project here.
+          </a>
+        </p>
+        {isAuthenticated ? (
+          <Link
+            to="/proposals"
+            className="button button--color-blue button--inverted"
+          >
+            <TableCellsIcon className="icon" />
+            Go to the Dashboard
+          </Link>
+        ) : (
+          <Button
+            color="blue"
+            inverted
+            onClick={() => { login('/').catch(logger.error); }}
+          >
+            <UserIcon className="icon" />
+            Log in
+          </Button>
+        )}
+        <dl>
+          {!isAuthenticated && (
+            <div className="dli">
+              <dt>Need an account?</dt>
+              <dd>
+                Email Jim McGowan:
```

Comment:
I don't know if we're in "move fast and break things" mode for this but this strikes me as a value that should be configurable / pulled from some kind of configuration file or environment variable.

Reply:
Since this is for a pilot group of like 20 people atm, all of whom know Jim, the clarity of this direction is actually preferred. As a point of process, we may want a more general inbox, but that's a decision out of scope of this implementation so I'm gonna move forward on this point!

I mean, I get that this could be `Email ${ACCOUNT_ADMIN_NAME}` and fair enough! But I'll reserve that for the first time we need to change it and/or have downtime before getting the pilot participant logins out this afternoon.

Reply:
@slifty My response here was initially written in response to [this comment](https://github.com/PhilanthropyDataCommons/data-viewer/pull/69#discussion_r1160888777) and then copied here with slight changes, so some of it actually addresses your point there.